.github/workflows/deploy-to-staging.yaml

```diff
@@ -6,10 +6,6 @@ on:
     branches: [main]
   workflow_dispatch:
     inputs:
-      deploy-app:
```

Comment:
Please check if any other workflow is dispatching onto this one using the `deploy-app` argument, and remove the arguments from the call if so.